src/app.js

```diff
@@ -1 +1,57 @@
 'use strict';
+
+const fs = require('fs');
+const readline = require('readline');
+
+const rl = readline.createInterface({
+  input: process.stdin,
+  output: process.stdout,
+});
```

Comment:
it would be possible not to use "readline.createInterface ", display errors through "throw Error", and the terminal closes automatically after copying.